internal/engine/eval/trusty/actions.go

```diff
@@ -428,8 +448,12 @@ func newSummaryPrHandler(
 	}, nil
 }
 
-func preprocessDetails(s string) string {
-	scanner := bufio.NewScanner(strings.NewReader(s))
+func preprocessDetails(s *string) string {
```

Comment:
Why change this to take a `*string`?  (If so, this looks like the correct behavior)